ethereumj-core/src/main/java/org/ethereum/util/RLP.java

```diff
@@ -694,6 +694,19 @@ private static void fullTraverse(byte[] msgData, int level, int startPos,
         }
     }
 
+    /**
+     * Compares supplied length information with maximum possible
+     * @param suppliedLength    Length info from header
+     * @param availableLength   Length of remaining object
+     * @throws RuntimeException if supplied length is bigger than available
+     */
+    private static void verifyLength(int suppliedLength, int availableLength) {
+        if (suppliedLength > availableLength) {
+            throw new RuntimeException(String.format("Length parsed from RLP (%s bytes) is greater " +
+                            "than remaining size of data (%s bytes)", suppliedLength, availableLength));
```

Comment:
I think it'd be better to read something like `than [probable | possible] size of data`